pages.fr/common/git.md

```diff
@@ -0,0 +1,20 @@
+# git
```

Comment:
We already have a `git.md` file. I don't really know why GitHub does not understand this, but this file should be removed from the PR since it's already there.